### ISSUE TITLE
Facebook: Doesn't load unread messages

### ISSUE THREAD

**Issue by seg-on** [miranda-ng/miranda-ng#1052]
Hi,
from last update stopped working load unread messages. I try load history and it doesn't work too.
```

CPU: Intel(R) Core(TM) i5-2410M CPU @ 2.30GHz [Intel64 Family 6 Model 42 Stepping 7] [DEP Enabled] [4 CPUs]
Installed RAM: 8127 MBytes
Microsoft Windows 10 Professional Edition, 64-bit (build 16299)
Internet Explorer: 9.11.16299.0 (build 916299)
Administrator privileges: No
OS Languages: (UI | Locale (User/System)) : Slovak/Slovak | Slovak/Slovak
Free disk space on Miranda partition: 109062 MBytes

Miranda NG Version: 0.95.8 alpha build #18910 (b6cb9d8d0d) x64
Build time: 5 Dec 2017 4:32:26
Profile: C:\Users\tomasv\AppData\Roaming\Miranda NG\segi\segi.dat (dbx mmap driver)
Profile size: 18743296 Bytes
Profile creation date: 18 Oct 2017 18:11:39
Language pack: Czech (CZ) [0405]
Service Mode: No

Active Plugins (26):
¤ AdvaImg.dll v.0.11.0.2 [4 Dec 2017 4:25:02] - Miranda image services 
¤ AVS.dll v.0.95.3.4 [4 Dec 2017 4:23:32] - Avatar service 
¤ Clist_nicer.dll v.0.9.2.4 [4 Dec 2017 4:25:34] - Clist nicer 
¤ Cln_skinedit.dll v.0.0.0.4 [4 Dec 2017 4:24:16] - Skin editor for Clist nicer 
¤ CrashDumper.dll v.0.0.5.2 [4 Dec 2017 4:29:18] - Crash dumper 
¤ DbEditorPP.dll v.4.0.1.2 [4 Dec 2017 4:29:34] - Database editor++ 
¤ Dbx_mmap.dll v.0.95.3.1 [5 Dec 2017 4:33:06] - Miranda NG mmap database driver 
¤ Facebook.dll v.0.3.3.2 [4 Dec 2017 4:26:44] - Facebook RM 
¤ Fingerprint.dll v.0.1.0.0 [4 Dec 2017 4:31:36] - Fingerprint NG 
¤ HistoryPP.dll v.1.5.1.5 [28 Nov 2017 11:33:10] - History++ 
¤ HistoryStats.dll v.0.2.1.1 [4 Dec 2017 4:33:56] - HistoryStats 
¤ Import.dll v.0.95.4.3 [4 Dec 2017 4:23:36] - Import contacts and messages 
¤ Jabber.dll v.0.11.0.5 [4 Dec 2017 4:29:50] - Jabber protocol 
¤ PluginUpdater.dll v.0.1.2.5 [4 Dec 2017 4:30:58] - Plugin updater 
¤ Popup.dll v.2.1.1.12 [4 Dec 2017 4:25:36] - Popup plus 
¤ QuickSearch.dll v.1.4.2.0 [18 Oct 2017 3:51:20] - Quick Search 
¤ SeenPlugin.dll v.5.0.5.3 [4 Dec 2017 4:25:48] - Last seen 
¤ SkypeWeb.dll v.0.12.2.5 [4 Dec 2017 4:34:02] - Skype protocol (Web) 
¤ SmileyAdd.dll v.0.3.0.1 [4 Dec 2017 4:27:14] - SmileyAdd 
¤ SpellChecker.dll v.0.2.6.4 [4 Dec 2017 4:30:48] - Spell checker 
¤ StatusManager.dll v.0.11.2.3 [4 Dec 2017 4:34:20] - Status manager 
¤ TabSRMM.dll v.3.5.0.7 [5 Dec 2017 4:34:42] - TabSRMM 
¤ Tipper.dll v.2.1.0.7 [4 Dec 2017 4:26:02] - Tipper 
¤ TopToolBar.dll v.0.8.0.4 [4 Dec 2017 4:26:58] - TopToolBar 
¤ UInfoEx.dll v.0.8.5.0 [4 Dec 2017 4:26:12] - User info ext 
¤ Weather.dll v.0.4.0.3 [4 Dec 2017 4:31:26] - Weather 

Unloadable Plugins (1):
¤ DbChecker.dll v.0.0.0.0 [5 Dec 2017 4:39:08] - <unknown> 
    Plugin statically linked to missing module: mir_core.mir
    Plugin statically linked to missing module: mir_app.mir


Weather ini files:
-------------------------------------------------------------------------------
 gismeteo.ini v.2012.05.03 [11 Oct 2016 16:18:02] - GisMeteo
 msn.ini v.2014-11-08 [11 Oct 2016 16:18:02] - MSN Weather (New Layout)
 weatherxml.ini v.0.0.5.3 2009-07-16 [11 Oct 2016 16:18:02] - WeatherXML
 wundergrnd_intl.ini v.2011/02/13 [11 Oct 2016 16:18:02] - Weather Underground Int'l
 wundergrnd_xml.ini v.2011/02/13 [11 Oct 2016 16:18:02] - Weather Underground XML

Protocols and Accounts:
-------------------------------------------------------------------------------
Facebook                 1 - Enabled 0 - Disabled  Loaded
JABBER                   4 - Enabled 1 - Disabled  Loaded
SKYPE                    1 - Enabled 0 - Disabled  Loaded
MetaContacts             1 - Enabled 0 - Disabled  Loaded
Weather                  1 - Enabled 0 - Disabled  Loaded

Icon Packs:
-------------------------------------------------------------------------------
 Avonlea Light.icl [21 Mar 2012 17:07:17]
 Flags_icons.dll [4 Dec 2017 4:40:22]
 Fp_icons.dll [4 Dec 2017 4:40:30]
 gtalk-gmailcoloredballs-v2.dll [7 Sep 2005 22:44:30]
 HistoryPP_icons.dll [28 Nov 2017 11:59:16]
 proto_ball1.dll [4 Sep 2005 15:52:06]
 Proto_Facebook.dll [4 Dec 2017 4:40:28]
 proto_gtalk.dll [24 Oct 2012 9:12:26]
 Proto_Jabber.dll [4 Dec 2017 4:40:26]
 Proto_MetaContacts.dll [5 Dec 2017 4:49:24]
 Proto_Skype.dll [4 Dec 2017 4:40:32]
 Proto_Weather.dll [4 Dec 2017 4:40:30]
 TabSRMM_icons.dll [5 Dec 2017 4:49:18]
 Toolbar_icons.dll [4 Dec 2017 4:40:24]
 UInfoEx_icons.dll [4 Dec 2017 4:40:22]
 xStatus_Jabber.dll [4 Dec 2017 4:40:28]
 zoho.dll [2 Feb 2017 17:57:19]
```

**Comment by zemiacsik** [comment]
Yep, they are not loading since Friday/Saturday night I think..

**Comment by Robyer** [comment]
This isn't regression in Miranda, it was caused by change on Facebook itself. Related to issue #1036 (same cause)

**Comment by georgehazan** [comment]
closed as duplicate of #1036
